### PR TITLE
PTDF with shift factors for fixed phase shifting transformers

### PR DESCRIPTION
**Pull request by anyacastillo**
This update is a bug fix that results in equivalent models for DCOPF and PTDF when there are fixed phase shifting transformers. The PTDF+losses model has also been updated to account for losses due to the voltage phase shift.

### CODE REVIEW

File: egret/model_library/transmission/branch.py [hunk 1]
```diff
@@ -423,6 +456,12 @@ def declare_eq_branch_loss_ptdf_approx(model, index_set, branches, bus_p_loads,
             for gen_name in gens_by_bus[bus_name]:
                 expr -= coef * m.pg[gen_name]
 
+            # for _, phi_loss in phi_loss_from.items():
+            #     expr += coef * phi_loss
+            #
+            # for _, phi_loss in phi_loss_to.items():
+            #     expr -= coef * phi_loss
+
```
Comment (bknueven) [inline egret/model_library/transmission/branch.py]: These lines (and 440 above it) being commented out seems a bit weird to me. I don't know enough about loss calculation to know what the right math is, so if they're not needed we should eliminate them and some of the setup above them.

Comment (anyacastillo) [reply egret/model_library/transmission/branch.py]: Thanks for catching this -- I updated the code

File: egret/model_library/transmission/branch.py [hunk 2]
```diff
@@ -409,10 +433,19 @@ def declare_eq_branch_loss_ptdf_approx(model, index_set, branches, bus_p_loads,
...
 
-        ptdf = branch['ldf']
-        for bus_name, coef in ptdf.items():
-            if ptdf_tol and abs(coef) < ptdf_tol:
-                coef = 0.
+        if branch['branch_type'] == 'transformer':
+            tau = branch['transformer_tap_ratio']
+            shift = math.radians(branch['transformer_phase_shift'])
+            g = tx_calc.calculate_conductance(branch)
+            #expr += (g/tau) * shift**2
```
Comment (bknueven) [inline egret/model_library/transmission/branch.py]: Should this also be uncommented?

Comment (anyacastillo) [reply egret/model_library/transmission/branch.py]: I uncommented it. I also fixed the nonzero reference angle issue in this PR.

File: egret/models/acopf.py [hunk 3]
```diff
@@ -502,14 +495,13 @@ def create_riv_acopf_model(model_data, include_feasibility_slack=False):
...
     ref_bus = md.data['system']['reference_bus']
-    model.vj[ref_bus].fix(0.0)
-    model.vr[ref_bus].setlb(0.0)
-
     ref_angle = md.data['system']['reference_bus_angle']
     if ref_angle != 0.0:
-        raise ValueError('The RIV ACOPF formulation currently only supports'
-                         ' a reference bus angle of 0 degrees, but an angle'
-                         ' of {} degrees was found.'.format(ref_angle))
+        print("IN HERE")
```
Comment (bknueven) [inline egret/models/acopf.py]: One final nit..

Comment (anyacastillo) [reply egret/models/acopf.py]: I've created a painful PR for myself. Last nit is gone :)

**Pull request merged**